Sources/BraintreeCoreSwift/BTHTTP.swift

```diff
@@ -30,42 +34,31 @@ import Security
     public var dispatchQueue: DispatchQueue = DispatchQueue.main
 
     // MARK: - Internal Properties
-    let cacheDateValidator: BTCacheDateValidator
+    let cacheDateValidator = BTCacheDateValidator()
     let baseURL: URL
-
-// TODO: - once BTAPIHTTP + BTGraphQLHTTP are converted to Swift, handle error cases for empty or nil credentials before API request is made
-    var authorizationFingerprint: String? = ""
-    var tokenizationKey: String? = ""
+    
+    // MARK: - Private Properties
+    private let authType: AuthorizationType
     
     // MARK: - Initializers
-
-    /// Initialize `BTHTTP` with the URL for the Braintree API
-    /// - Parameter url: The base URL for the Braintree Client API
-    @objc(initWithBaseURL:)
-    public init(url: URL) {
-        self.baseURL = url
-        self.cacheDateValidator = BTCacheDateValidator()
-        self.pinnedCertificates = BTAPIPinnedCertificates.trustedCertificates()
-    }
```

Comment:
this init was never used, so I removed it and removed `convenience` from the two other inits.